pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchema.java

```diff
@@ -86,26 +86,46 @@ private KeyValueSchema(Schema<K> keySchema,
         this.schemaInfo.setSchema(byteBuffer.array());
     }
 
-    // encode as bytes: [key.length][key.bytes][value.length][value.bytes]
+    // encode as bytes: [key.length][key.bytes][value.length][value.bytes] or [value.length][value.bytes]
+    // if keyIsStoredToMessage is true, message.getKey() will be saved. Default keyIsStoredToMessage is null.
```

Comment:
”if keyIsStoredToMessage is true“ == > "keyIsStoredToMessage is a String, if keyIsStoredToMessage is equals true".